docusaurus/docs/shared/create-plugin-frontend.md

```diff
@@ -0,0 +1,39 @@
+The Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin) is a CLI application that simplifies Grafana plugin development, so that you can focus on code. The tool scaffolds a starter plugin, all the required configuration, and a development environment using [Docker Compose](https://docs.docker.com/compose/) for you.
+
+1. <span>In a new directory, create a plugin from a template using create-plugin. When prompted for the kind of plugin, select {props.pluginType}</span>:
```

Comment:
```suggestion
1. In a new directory, create a plugin from a template using the create-plugin tool. When prompted for the kind of plugin, select `{props.pluginType}`:
```

Reply:
<span> is required for this value to correctly be substituted